cmd/utils/flags.go

```diff
@@ -757,14 +762,23 @@ func SetupNetwork(ctx *cli.Context) {
 		NetworkIdFlag.Value = 0
 		core.BlockReward = big.NewInt(1.5e+18)
 		core.ExpDiffPeriod = big.NewInt(math.MaxInt64)
+	default:
+		params.TargetGasLimit = common.String2Big(ctx.GlobalString(TargetGasLimitFlag.Name))
```

Comment:
Why is this on the `default` branch and not executed for all? I.e. I know Olympic's dead now, but I don't see why we couldn't enable target gas for Olympic nonetheless